app/main.py

```diff
@@ -1,10 +1,9 @@
 def copy_file(command: str) -> None:
     try:
-        cp, main_file, for_copy_file = command.split()
+        command, main_file, for_copy_file = command.split()
     except ValueError:
         return
-
-    if main_file == for_copy_file:
+    if command != "cp" or main_file == for_copy_file:
         return
```

Comment:
```suggestion
if command == "cp" and main_file == for_copy_file:
    <your other code>
```

Try to not finish code like this, I mean raw returns are not a good sign

Reply:
Fixed that